src/apps/bristleback_apps/bm_rbr/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ set(LIB_FILES
     ${SRC_DIR}/lib/common/serial.c
     ${SRC_DIR}/lib/common/serial_console_u5.cpp
     ${SRC_DIR}/lib/common/stress.c
+    # ${SRC_DIR}/lib/common/trace.c
```

Comment:
to have the ability to enable ISR tracing this file needs to be included. I left it commented out, but maybe we can have a cmake define that protects/enables this?

Reply:
Can you put a comment like "Enable for ISR timing debugging" next to it?

Reply:
Done